docs/user/_sidebar.md

```diff
@@ -1,13 +1,17 @@
-* [Home](/)
-* [Kyma Telemetry Module](README.md)
-* [Telemetry Manager](01-manager.md)
-* [Gateways](gateways.md)
-* [Application Logs](02-logs.md)
-* [Traces](03-traces.md)
-* [Metrics](04-metrics.md)
-* [Tutorials](05-tutorials.md)
-* [Resources](resources/README.md)
-  * [Telemetry](01-telemetry.md)
-  * [LogPipeline](02-logpipeline.md)
-  * [TracePipeline](04-tracepipeline.md)
-  * [MetricPipeline](05-metricpipeline.md)
+- [Home](/)
+- [Kyma Telemetry Module](README.md)
+- [Telemetry Manager](01-manager.md)
+- [Gateways](gateways.md)
+- [Application Logs](02-logs.md)
+- [Traces](03-traces.md)
+- [Metrics](04-metrics.md)
+- [Integration Guides](integration/README.md)
+  - [OpenTelemetry Demo App](./integration/opentelemetry-demo/README.md)
+  - [SAP Cloud Logging](./integration/cloudlogging/README.md)
```

Comment:
you removed SAP from the file name but kept it in title, maybe add it to filename again?